src/components/PastFiles.js

```diff
@@ -1,5 +1,126 @@
+import { Icon } from '@iconify/react';
+import ReactJson from 'react-json-view';
+import { useState } from 'react';
+
+const files = [
+  {
+    name: 'bundle1.json',
+    size: '500 KB',
+    dateAdded: '10/10/2022',
+    body: {
+      resourceType: 'StructureDefinition',
+    },
+  },
+  {
+    name: 'bundle2.json',
+    size: '500 KB',
+    dateAdded: '10/10/2022',
+    body: {
+      resourceType: 'StructureDefinition',
+    },
+  },
+  {
+    name: 'bundle3.json',
+    size: '500 KB',
+    dateAdded: '10/10/2022',
+    body: {
+      resourceType: 'StructureDefinition',
+    },
+  },
+];
+
 function PastFiles() {
-  return <div />;
+  const [jsonViewIndexes, setJsonViewIndexes] = useState(new Set());
+  // I don't think we'll need this state in prod, pastFiles and setPastFiles
+  // should be passed in as props
+  const [pastFiles, setPastFiles] = useState(files);
+
+  function toggleView(index) {
+    const modifiedViewIndexes = new Set(jsonViewIndexes);
+    if (jsonViewIndexes.has(index)) {
+      modifiedViewIndexes.delete(index);
+    } else {
+      modifiedViewIndexes.add(index);
+    }
+    setJsonViewIndexes(modifiedViewIndexes);
+  }
+
+  function handleDelete(index) {
+    setPastFiles([...pastFiles.slice(0, index), ...pastFiles.slice(index + 1)]);
+  }
+
+  return (
+    <>
+      <h1 style={{ fontSize: '32px', fontWeight: 'bold' }}>Past Files</h1>
```

Comment:
DP1 - Let's replace this with tailwind styles. Here's a convenient reference sheet: https://nerdcave.com/tailwind-cheat-sheet

Reply:
Swapped both properties over to equivalent tailwind classes in commit [665b9b7](https://github.com/mcode/mcode-coverage-checker/pull/20/commits/665b9b7e798824bb66e55b199805a8995958adf0).